Arena.toml

```diff
@@ -1875,6 +2065,11 @@ document = "stjudecloud/workflows:/tools/kraken2.wdl"
 message = "kraken2.wdl:321:13: note[TrailingComma]: item missing trailing comma"
 permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/kraken2.wdl/#L321"
 
+[[diagnostics]]
+document = "stjudecloud/workflows:/tools/kraken2.wdl"
+message = "kraken2.wdl:330:14: note[DisallowedInputName]: declaration identifier must be at least 3 characters"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/kraken2.wdl/#L330"
```

Comment:
`db` feels similar to `id` to me. A well known abbreviation. Not sure what the right call is.

Reply:
`db` feels similar in that you can use `database` or `except`.